src/instructlab/data/generate.py

```diff
@@ -125,6 +125,15 @@
     "--model-family",
     help="Force model family to use when picking a generation template",
 )
+@click.option(
+    "--pipeline",
+    type=click.Choice(["simple", "full"], case_sensitive=False),
```

Comment:
Why do we add `full` if it's not implemented?

Reply:
it's implemented in `instructlab-sdg` main and I want to be able to test it